app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Admin::UsersController < ApplicationController
```

Comment:
will validate admin execute on this controller actions?

Reply:
No, it will not. 
Have fixed it.